src/main/java/training/App.java

```diff
@@ -8,7 +8,66 @@ public String getGreeting() {
         return "Hello consortium world.";
     }
 
-    public static void main(String[] args) {
+    public static void main(String[] args) { Rectangle r = new Rectangle(10, 4);
+        Square s = new Square(7);
+        Circle c = new Circle(3.5);
+
+        System.out.println("Rectangle Area : " + r.getArea());
+        System.out.println("Square Area : " + s.getArea());
+        System.out.println("Circle Area : " + c.getArea());
+        System.out.println();
+
         System.out.println(new App().getGreeting());
+
+    }
+}
+abstract class Shapes
```

Comment:
This is good I was expecting it to be like this as shows how inheritance should be used